templates/autoscale-existing-vpc.template

```diff
@@ -0,0 +1,1049 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "Fortinet FortiGate Auto Scaling [existing VPC]. This template deploys Fortinet FortiGate Auto Scaling into an existing VPC. For more details, please see the Fortinet FortiGate Auto Scaling documentation. **WARNING** You will be billed for the FortiGate instances and related AWS resources if you create a stack from this template.",
+    "Parameters": {
+        "ResourceTagPrefix": {
+            "Type": "String",
+            "AllowedPattern": "[0-9a-zA-Z@.#-]+",
+            "MaxLength": 50,
+            "ConstraintDescription": "Resource tag prefix can include numbers, lowercase letters, uppercase letters, ampersat(@) , hyphens (-), period (.), and hash (#). Max length is 50.",
+            "Description": "The ResourceGroup Tag Key used on all resources and as the name prefix of all applicable resources. Can only contain numbers, lowercase letters, uppercase letters, ampersat(@), hyphens (-), period (.), and hash (#). Max length is 50."
```

Comment:
```subscription
            "Description": "The ResourceGroup Tag Key used on all resources and as the name prefix of all applicable resources. It can only contain numbers, lowercase letters, uppercase letters, ampersat(@), hyphens (-), period (.), and hash (#). Max length is 50."
```